# @file        measures.py
# @author      Jacobo Costas
# @version     1.0
# @date        Created:     16/3/2016
# ------------------------------------------------------------------------
# Description: Formato de mensaje utilizado para cualquier metodo calculo.
# ------------------------------------------------------------------------
# History:     1.0 Inicial
# ------------------------------------------------------------------------
Header header
uint8[] image
# Valor del semieje
float32 major_axis
float32 minor_axis

float32 orientation
float32 x
float32 y
